jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/RuntimeDataServiceImplTest.java

```diff
@@ -1016,12 +1017,15 @@ public void testGetTaskByWorkItemId() {
     	assertNotNull(userTask);
     	assertEquals(processInstanceId, userTask.getProcessInstanceId());
     	assertEquals("Write a Document", userTask.getName());
+        assertTask(userTask, processInstanceId);
 
     }
 
     @Test
     public void testGetTaskById() {
```

Comment:
Can you create a new test for this as this is modifying the old one? It uses a different startProcess method.

Reply:
I think once the name of asserTask method has been changed to assertCorrelationAndProcess, it is now clear the pupose of adding this line here. To test that for getTaskByWorkItemId, processType and correlationKey are included in the response

Reply:
I mean the test which I commented `testGetTaskById()` :) As you changed the behaviour of the test - not that good for regression testing. (For example you use now a different startProcess method) So I would say that a separate test would be better.

Reply:
Ok, understood ;)
New test added.